src/server/src/routes/job.ts

```diff
@@ -0,0 +1,71 @@
+import StatusCodes from 'http-status-codes';
+import { Request, Response, Router } from 'express';
+import { IJob } from '@entities/job';
+import { errors } from '@shared/errors';
+import {
+    createJob,
+} from '@modules/job';
+import logger from '@shared/Logger';
+
+const router = Router();
+
+const { BAD_REQUEST, CREATED, OK, INTERNAL_SERVER_ERROR } = StatusCodes;
+
+interface jobRequest extends Request {
+    body: {
+        job: IJob;
+    };
+}
+
+/******************************************************************************
+ *            POST Request example - Create - "POST /api/job/create"
+ ******************************************************************************/
+
+router.post('/create', async (req: jobRequest, res: Response) => {
+    const { job } = req.body;
+    const {
+        targetYears,
+        hoursPerWeek,
+        description,
+        expirationDate,
+        startDate,
+        endDate,
+        type,
+        title,
+        status,
+        minSalary,
+        maxSalary,
+        departmentId } = job;
+    if (!job) {
+        return res.status(BAD_REQUEST).json({
+            error: errors.paramMissingError,
+        });
+    }
```

Comment:
I think we need to add more validation here. For example if the user forgets to send the description or title then the request should fail since those fields are mandatory. Try something like this:

if (!description || !title || ...  !<other required attributes>){
    return res.status(BAD_REQUEST).json({
            error: errors.paramMissingError,
        });
}

Reply:
Done.